kratos/includes/mortar_classes.h

```diff
@@ -1307,8 +1307,8 @@ class DualLagrangeMultiplierOperators
         )
     {
         /* DEFINITIONS */
-        const Vector n1 = rKinematicVariables.NSlave;
-        const double det_j = rKinematicVariables.DetjSlave; 
+        const Vector& n1 = rKinematicVariables.NSlave;
+        const double& det_j = rKinematicVariables.DetjSlave; 
         
         De += rIntegrationWeight * (ComputeDe(n1, det_j));
```

Comment:
what is "ComputeDe"? make the name more explicit (read longer, so that one understands what it does)

Reply:
Computes the matrix De, is a matrix that is part of the Dual LM formulation